mods/ctf/ctf_map/ctf_traps.lua

```diff
@@ -101,3 +101,101 @@ minetest.register_node("ctf_map:reinforced_cobble", {
 	groups = {cracky = 1, stone = 2},
 	sounds = default.node_sound_stone_defaults(),
 })
+
+minetest.register_node("ctf_map:landmine", {
+	description = "Landmine",
+	drawtype = "nodebox",
+	tiles = {"ctf_map_landmine.png", "ctf_map_landmine.png^[transformFY"},
+	inventory_image = "ctf_map_landmine.png",
+	paramtype = "light",
+	sunlight_propagates = true,
+	walkable = false,
+	groups = {cracky=1, level=2},
+	node_box = {
+		type = "fixed",
+		fixed = {-0.5, -0.5, -0.5, 0.5, -0.4, 0.5},
+	},
+	selection_box = {
+		type = "fixed",
+		fixed = {-0.5, -0.5, -0.5, 0.5, -0.4, 0.5},
+	},
+	after_place_node = function(pos, placer, itemstack, pointed_thing)
+		local meta = minetest.get_meta(pos)
+		local name = placer:get_player_name()
+
+		meta:set_string("placer", minetest.serialize({
+			team = ctf_teams.get(name),
+			name = name,
+		}))
+	end
+})
+
+minetest.register_abm({
+	label = "Landmine",
+	nodenames = {"ctf_map:landmine"},
+	interval = 1,
+	chance = 1,
+	min_y = -3000,
```

Comment:
```suggestion
	min_y = 0,
```
I don't think maps are placed below y=0